### ISSUE TITLE
Bug Report:  setting ULF truncates logging.properties /FISH-5654

### ISSUE THREAD

**Issue by donsizemore** [payara/Payara#5368]
# Description #
----------

In Payara-5.2021.5, issuing the command:

`/usr/local/payara5/bin/asadmin set-log-file-format ULF`

truncates logging.properties from 8174 bytes to 65, containing the two lines:

```
#GlassFish logging.properties list
#Tue Aug 03 19:53:57 UTC 2021
```

## Expected Outcome ##

I should expect this command to set the log file format to ULF and carry on.

## Current Outcome ##

logging.properties is truncated from 8174 bytes to 65, containing only two commented-out lines. Further, Payara attempts to update logging.properties in this way approximately twice per second, filling server.log with a bunch of INFO statements to this effect.

## Steps to reproduce ## 

Verify size and contents of logging.properties. Issue:

`/usr/local/payara5/bin/asadmin set-log-file-format ULF`

as the payara service account user, and see that logging.properties now contains only two commented-out lines.

## Environment ##

- **Distribution**: Server Full Profile
- **JDK Version**: java-11-openjdk-11.0.12.0.7-0.el8_4.x86_64
- **Operating System**: RockyLinux

**Comment by rdebusscher** [comment]
Hi,

Some refactoring introduced a bug in 5.2021.5 on JDK 11 when setting the logging attributes through the Admin Console/asadmin commands due to JDK 8/JDK 11 internal differences within Properties Class.  This will be fixed in the next release.

**Comment by donsizemore** [comment]
@rdebusscher Thank you!

**Comment by donsizemore** [comment]
This is indeed fixed in 5.2021.6.

Thank you!
